Configurations/00-base-templates.conf

```diff
@@ -92,18 +89,14 @@ my %targets=(
         lib_defines      =>
             sub {
                 my @defs = ();
-                unless ($disabled{"zlib-dynamic"}) {
-                    my $zlib = $withargs{zlib_lib} // "ZLIB1";
-                    push @defs, 'LIBZ=' . (quotify("perl", $zlib))[0];
-                }
+                my $zlib = $withargs{zlib_lib} // "ZLIB1";
+                push @defs, 'LIBZ=' . (quotify("perl", $zlib))[0];
                 return [ @defs ];
             },
         ex_libs         =>
             sub {
-                unless ($disabled{zlib}) {
-                    if (defined($disabled{"zlib-dynamic"})) {
-                        return $withargs{zlib_lib} // "ZLIB1";
-                    }
+                if (defined($disabled{"zlib-dynamic"})) {
+                    return $withargs{zlib_lib} // "ZLIB1";
                 }
```

Comment:
This doesn't look right.  It should be this:


``` perl
        ex_libs         =>
            sub {
                unless ($disabled{zlib}) {
                    return $withargs{zlib_lib} // "ZLIB1";
                }
                return ();
            },

Reply:
I also made this change, updated commit pushed.  Thanks!